risclient/risclient.go

```diff
@@ -23,6 +25,7 @@ type RISClient struct {
 	req    *Request
 	cc     *grpc.ClientConn
 	c      Client
+	b      *backoff.ExponentialBackOff
```

Comment:
I see we're having a bunch of very short names here. However, I think just be is not very descriptive. Let's rename it to backoff.